### PR TITLE
Require readme's for our projects

### PR DESCRIPTION
**Pull request by oponder**
I'd like for all our projects to have a Description and README. Do we have this agreement in writing anywhere already?

It doesn't have to be super long, but something that helps people get some context about what the project is for.

Triggered by stumbling into bridge-operator: https://github.com/giantswarm/bridge-operator

### CODE REVIEW

File: github/readmes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Project Readme and Description
```
Comment (kopiczko) [inline github/readmes]: Please add `.md` suffix to the file and link it in the ToC in the main README.md of this project.

Comment (oponder) [reply github/readmes]: Thanks, done!

**Review by MarcelMue — APPROVED** [review]

**Review by kopiczko — APPROVED** [review]

File: github/readmes.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Project Readme and Description
+
+Projects must have a description and a readme that explains what it is for and where it fits in to the big picture.
+
+Some indication of what other services it collaborates with, what kind of storage, data structures or CRD's it depends on would be good to have in the README as well.
+
+Depending on how 'far up the stack' they are, they can have usage examples.
+
+If it is a tool that we make public, then usage examples are a must.
```
Comment (xh3b4sd) [inline github/readmes.md]: You have an example for the usage example? I would not know how to write that now.

File: github/readmes.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Project Readme and Description
+
+Projects must have a description and a readme that explains what it is for and where it fits in to the big picture.
```
Comment (xh3b4sd) [inline github/readmes.md]: Breaking lines at 80 characters or something like this would be really grand.

**Review by xh3b4sd — APPROVED** [review]

**Review by rossf7 — APPROVED** [review]

File: github/readmes.md (hunk 2 above)
Comment (oponder) [reply github/readmes.md]: @xh3b4sd added some text, does that help?

Comment (xh3b4sd) [reply github/readmes.md]: Yes that is cool. Thank you.

**Pull request merged**